ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/CommandStoreUtil.java

```diff
@@ -40,7 +40,7 @@ public static void httpWaitForCommandSequenceNumber(
       final long seqNum = request.getCommandSequenceNumber().orElse(-1L);
       final String errorMsg = "Interrupted while waiting for command with the supplied "
           + "sequence number to execute. sequence number: " + seqNum
-          + ", request: " + request.getKsql();
+          + ", request: " + request.getMaskedKsql();
```

Comment:
Per comment above, we definitely need a defensive check-and-set for this field here.